clair/clair.go

```diff
@@ -58,8 +59,23 @@ func (imageAnalysis ImageAnalysis) ShortName(l v1.Layer) string {
 //Config configure Clair from configFile
 func Config() {
 	uri = fmtURI(viper.GetString("clair.uri"), viper.GetInt("clair.port")) + "/v1"
-	host = viper.GetString("clair.host")
 	healthURI = fmtURI(viper.GetString("clair.uri"), viper.GetInt("clair.healthPort")) + "/health"
 	Report.Path = viper.GetString("clair.report.path")
 	Report.Format = viper.GetString("clair.report.format")
+
+	headersArray := viper.GetStringSlice("clair.headers")
```

Comment:
I would use clair.request.headers

Dur to the reverse proxy thing, we could need response header too